frontend/src/lib/Buttons.ts

```diff
@@ -176,6 +176,30 @@ export default class Buttons {
     return this;
   }
 
+  public deletePipelineVersionV2(
```

Comment:
Similar question to the what I asked on `PipelinesDialogV2`, do we need both v1 and v2 mehtods, when and where would the v1 button be used?

Reply:
This is solved after merging the logic of dialogHandler.